decidim-core/app/views/layouts/decidim/_social_meta.html.erb

```diff
@@ -1,3 +1,9 @@
+<% content_for :meta_description, strip_tags(translated_attribute(current_organization.description)), flush: true %>
```

Comment:
If these are the default values, shouldn't we flush in the other partials instead of here?